### PR TITLE
Adds in the necessary files to build in new pip version

### PR DESCRIPTION
**Pull request by owencjones**

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -1 +1 @@
-numpy
+numpy<=1.23.0
```
Comment (simedw) [inline requirements-test.txt:1]: Why is this necessary?

Comment (owencjones) [reply requirements-test.txt:1]: https://numpy.org/doc/stable/release/1.24.0-notes.html

**Review by simedw — APPROVED** [review]
LGTM, but let's refactor the code so we can start using newer version of numpy soon.

**Comment by owencjones** [comment]
Many thanks @simedw - See ticket IO-714 for task, will try to address in next couple of cycles.
